MQuince/MQuince.Integration.Services.Implementation/SftpService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using MQuince.Integration.Entities;
+using MQuince.Integration.Services.Constracts.Interfaces;
+using Renci.SshNet;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace MQuince.Integration.Services.Implementation
+{
+    public class SftpService : ISftpService
+    {
+        public bool SendFile(string fileName)
```

Comment:
This is a good place for refactoring in couple of methods. 
Throwing exception when client is not connected and handling in controller is better way of telling if the operation was succesfull. Then you can change return type to void.